### PR TITLE
update requirements

### PR DESCRIPTION
**Pull request by BoZhaoUTSC**
Fixes Issue #

- [ ] Patch: Bug Fix?
- [ ] Minor: New Feature?
- [ ] Major: Breaking Change?
- [x] Documentation?
- [ ] New dependencies?

### Other Notes
- remove support of multi courses
- rename ADMIN to INSTRUCTOR

### CODE REVIEW

File: docs/REQUIREMENTS.md [hunk 1]
```diff
@@ -71,24 +59,22 @@
...
-- Add annotation to a review  
+- Review designated review submission
+- Add annotation to a review
 
 ### I should be able to:
-- Add a grade to an assignment?
-- Get a list of all the courses?  
-- Enroll in a course?
+- Add a grade to an assignment
+- Select a theme of syntax highlighter
```
[the remark this answers is not in the document]
Comment (BoZhaoUTSC) [reply docs/REQUIREMENTS.md]: Add theme to entire application is gonna be hard.

File: docs/REQUIREMENTS.md [hunk 2]
```diff
@@ -3,35 +3,30 @@
...
-- Admin
+- INSTRUCTOR
 - Student
 
 ## As a User
 ### I must be able to:
 - Login and authenticate myself
 - Log out
-## As a Admin
+## As an INSTRUCTOR
```
[the remark this answers is not in the document]
Comment (BoZhaoUTSC) [reply docs/REQUIREMENTS.md]: > They should be able to update the course details

what are course details?

[the remark this answers is not in the document]
Comment (Mystaken) [reply docs/REQUIREMENTS.md]: Any general info related to a course. (title, course code, website, etc)

File: docs/REQUIREMENTS.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Mystaken) [reply docs/REQUIREMENTS.md]: We will always have at least one theme. If we aren't going to add themeing, then we will just have one theme. But, I think that the syntax highlighter should be consistent with the rest of the application and I don't think we need to have an option to change the themes of specific components.

[the remark this answers is not in the document]
Comment (BoZhaoUTSC) [reply docs/REQUIREMENTS.md]: > We will always have at least one theme. If we aren't going to add themeing, then we will just have one theme. But, I think that the syntax highlighter should be consistent with the rest of the application and I don't think we need to have an option to change the themes of specific components. (i.e. either we change the theme of all the components, or we change none of them)

Got it.

**Pull request merged**